src/CodeGen_C.cpp

```diff
@@ -108,6 +108,12 @@ inline float float_from_bits(uint32_t bits) {return reinterpret<float, uint32_t>
 template<typename T> T max(T a, T b) {if (a > b) return a; return b;}
 template<typename T> T min(T a, T b) {if (a < b) return a; return b;}
 
+template<typename A, typename B>
```

Comment:
If A or B had no copy ctors then this would still allow it to work. Probably a nonissue here since we probably never generate things that don't have them.